IPython/core/display.py

```diff
@@ -475,8 +475,13 @@ class Latex(TextDisplayObject):
 
     def _repr_latex_(self):
         return self.data
+        
+        
+class MathML(TextDisplayObject):
```

Comment:
I was thinking of the latter since `Math` currently only renders LaTeX and if we now support other mathematical notation formats like MathML, it would imply that `Math` also render them.